src/HeaderUserInformation/HeaderUserInformation.tsx

```diff
@@ -36,8 +36,21 @@ export interface HeaderUserInformationProps {
   username: string;
   id?: UUID;
   isLoading?: boolean;
+  isPopUpOpen?: boolean;
   noUsernameMessage?: string;
   onClick?: MouseEventHandler<HTMLDivElement>;
+  popUpType?:
+    | boolean
+    | 'dialog'
+    | 'menu'
+    | 'grid'
+    | 'listbox'
+    | 'false'
+    | 'true'
+    | 'tree'
+    | undefined;
```

Comment:
Not sure why the popup type is manually enumerated here? Is this derived from an existing type? Also you don't need to specify `| undefined` if the prop is marked as optional with the `?` annotation.